build.gradle

```diff
@@ -9,12 +12,13 @@ Set testModules = [
 subprojects {
 
     group = 'com.paypal.mocca'
-    version = '0.0.3'
+    version = '0.0.4-SNAPSHOT'
```

Comment:
This is NOT supposed to be done in master branch, it is done in develop branch (it is done already).

Reply:
I again issued PR to master branch. Will close and open it on develop